styles/page.css

```diff
@@ -0,0 +1,56 @@
+/*Load Colors from colors.css*/
+@import "colors.css";
+
+/* Normalize */
+* {
+    margin: 0;
+    padding: 0;
+    /* background-color: var(--backgroundColor); */
+}
+
+/* General Elements Styling */
+body {
+    font-family: 'Rajdhani', sans-serif;
+    margin-top: 5rem;
+    background-color: #f0ece7;
+    color: black;
+    width: 100%;
+    font-size: 1.8rem;
+}
+/* End of General Elements Styling */
+
+/* Page Identifier Name */
+main {
```

Comment:
should main be moved to general elements styling? I wrote it here at that time but can other people share their opinons

Reply:
Yes it should be moved to general styling.